src/test/ui/proc-macro/capture-unglued-token.stdout

```diff
@@ -1,5 +1,4 @@
-PRINT-BANG INPUT (DISPLAY): Vec<u8>
-PRINT-BANG RE-COLLECTED (DISPLAY): Vec < u8 >
+PRINT-BANG INPUT (DISPLAY): Vec < u8 >
```

Comment:
This is expected because token pretty-printer is now used instead of AST pretty-printer.